staging/src/k8s.io/kms/internal/plugins/_mock/pkcs11/pkcs11.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pkcs11
+
+import (
+	"context"
+	"crypto/cipher"
+	"crypto/rand"
+	"fmt"
+
+	crypot11 "github.com/ThalesIgnite/crypto11"
+
+	"k8s.io/kms/pkg/service"
+)
+
+const (
+	mockAnnotationKey = "version.encryption.remote.io"
+)
+
+var _ service.Service = &pkcs11RemoteService{}
+
+type pkcs11RemoteService struct {
+	key    *crypot11.SecretKey
+	keyID  string
+	cipher cipher.AEAD
```

Comment:
nit, just to be consistent with the field name used in k/k

```suggestion
	aead cipher.AEAD
```